lib/src/transformer_implementation.dart

```diff
@@ -3774,6 +3973,7 @@ initializeReflectable() {
       if (const bool.fromEnvironment("reflectable.pause.at.exit")) {
         _processedEntryPointCount++;
       }
+      _resolver.release();
```

Comment:
Could we wrap this code in a local function, so we only have to release the resolver in one spot?

Reply:
Reorganized the code without abstractions, because they seemed to require a non-local return. See https://dartpad.dartlang.org/e6351d655334455c511288345c5fbaec for details.